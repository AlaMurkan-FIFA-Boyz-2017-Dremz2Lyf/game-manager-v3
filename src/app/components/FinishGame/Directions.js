import React from 'react';
import { Segment, Header, Icon, Message } from 'semantic-ui-react';

const Directions = () => (
  <Segment basic textAlign="center">
    <Header color="teal" as="h2">
      <Icon name="share" />
      <Header.Content>
        Directions
      </Header.Content>
    </Header>
    <Message color="teal">
      Please select Match Facts in the post-match menu. When there, please use the share feature to take a screenshot.
    </Message>
  </Segment>
);

export default Directions;
