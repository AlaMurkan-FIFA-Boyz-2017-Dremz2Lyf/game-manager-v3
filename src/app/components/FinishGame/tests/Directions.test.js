import React from 'react';
import { shallow } from 'enzyme';

import Directions from '../Directions';

describe('Directions Component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Directions />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
