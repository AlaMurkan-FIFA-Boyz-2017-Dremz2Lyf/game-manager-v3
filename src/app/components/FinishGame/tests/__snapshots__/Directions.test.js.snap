// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Directions Component should match snapshot 1`] = `
<Segment
  basic={true}
  textAlign="center"
>
  <Header
    as="h2"
    color="teal"
  >
    <Icon
      as="i"
      name="share"
    />
    <HeaderContent>
      Directions
    </HeaderContent>
  </Header>
  <Message
    color="teal"
  >
    Please select Match Facts in the post-match menu. When there, please use the share feature to take a screenshot.
  </Message>
</Segment>
`;
